packages/harmony/src/assets/styles/spacing.css

```diff
@@ -9,6 +9,15 @@
 root {
   --harmony-unit: 4px;
 
+  --harmony-spacing-xs: 4px;
```

Comment:
And so I'm clear this is cause buttons and other elements may need to be on increments of four, but when users use harmony spacing, they would use the unit system?

Reply:
Yeah the units are for stuff like heights and absolute sizes whereas spacing is for gaps between stuff. I could see using raw px values for sizes though.